<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>KShortcutDialogSimple</class>
 <widget class="QWidget" name="KShortcutDialogSimple" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>38</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>Shortcut:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" >
    <widget class="KPushButton" name="m_txtShortcut" >
     <property name="cursor" >
      <cursor>4</cursor>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="m_btnClearShortcut" >
     <property name="toolTip" >
      <string>Clear shortcut</string>
     </property>
     <property name="text" >
      <string>x</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QWidget</extends>
   <header>kpushbutton.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
